src/sentry/models/organizationmembermapping.py

```diff
@@ -53,7 +53,7 @@ class Meta:
         )
 
     def save(self, *args, **kwds):
-        with transaction.atomic(), in_test_psql_role_override("postgres"):
+        with outbox_context(transaction.atomic()):
```

Comment:
Not directly related to these changes, but having both orgmember and orgmembermapping emit outboxes feels like it could become a risk in the future. We could create a scenario where an update in member triggers a membermapping change. When that membermapping change occurs, the data is pushed to the region and the member is updated. This sequence of events could become cyclic if the update payloads from both outbox messages contains overlapping attributes. If you share this concern, I can add a test to help prevent it from being introduced.

Reply:
So interestingly, organizationmembermapping does not emit outboxes for itself, but rather for the user associated with it.  These outboxes essentially are required to ensure that denormalized user data follows all changes to membership within regions.

Reply:
OH!  Sorry, so this is guarded because the outboxes ONLY create for new memberships.  It can't be a loop due to restriction on create, not update.

Reply:
However, I think you are right, I'm going to move this logic directly into the service for adding memberships.